### PR TITLE
#2693 - Added link for product tab review

### PR DESCRIPTION
**Pull request by teodormoq**

### CODE REVIEW

**Comment by Elena-Evsa** [comment]
Oops page appears then click on 'Be the first to review this product'
https://vimeo.com/577996099/f3fbe6801e

File: packages/scandipwa/src/route/ProductPage/ProductPage.component.js [hunk 1]
```diff
@@ -56,7 +57,8 @@ export class ProductPage extends PureComponent {
         isInformationTabEmpty: PropTypes.bool.isRequired,
         isAttributesTabEmpty: PropTypes.bool.isRequired,
         selectedBundlePriceExclTax: PropTypes.number.isRequired,
-        selectedInitialBundlePrice: PropTypes.number.isRequired
+        selectedInitialBundlePrice: PropTypes.number.isRequired,
+        ProductTabsDefaultValue: PropTypes.number.isRequired
```
Comment (ejnshtein) [inline packages/scandipwa/src/route/ProductPage/ProductPage.component.js:61]: Probably we should use camelCase for this prop, as everywhere else to keep our code consistent.

**Review by ejnshtein — CHANGES_REQUESTED** [review]
Resolve case issue, please!

**Comment by teodormoq** [comment]
Hi @ejnshtein i dont have time to fix this sorry

**Comment by carinadues** [comment]
PR closed for the reasons explained in the issue.

**Pull request closed without merging**